ratelimitj-redis/src/test/java/es/moki/ratelimitj/redis/request/RedisSlidingWindowSyncRequestRateLimiterTest.java

```diff
@@ -42,4 +52,33 @@ void afterEach() {
     protected RequestRateLimiter getRateLimiter(Set<RequestLimitRule> rules, TimeSupplier timeSupplier) {
         return new RedisSlidingWindowRequestRateLimiter(connect, rules, timeSupplier);
     }
+    
+
+    @Test
+    void shouldCheckIncrementAnyway() throws Exception {
+    	
+        ImmutableSet<RequestLimitRule> rules = ImmutableSet.of(RequestLimitRule.of(20, TimeUnit.SECONDS, 5));
+        TimeBanditSupplier timeBandit = new TimeBanditSupplier();
+        
+        RequestRateLimiter requestRateLimiter = getRateLimiter(rules, timeBandit);
+
+        String key = "ip:127.0.0.5";
+
+        timeBandit.addUnixTimeMilliSeconds(100L);
+        // Counter should be 0 now
+        assertThat(requestRateLimiter.overLimitWhenIncremented(key, 1)).isFalse();
+        // Counter should be 1 now and 'false', meaning not over limit
+        LOG.debug("About to print add 10 to go over.");
```

Comment:
IMHO the log debug messages are not needed, and they could be Java comments if they bring any value